### PR TITLE
Reload WebXR sites when settings are changed

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/PrivacyOptionsView.java [hunk 1]
```diff
@@ -323,6 +320,9 @@ private void setWebXR(boolean value, boolean doApply) {
 
         if (doApply) {
             SettingsStore.getInstance(getContext()).setWebXREnabled(value);
+            for (WindowWidget window: mWidgetManager.getWindows().getCurrentWindows()) {
+                window.getSession().reload(GeckoSession.LOAD_FLAGS_NONE);
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/PrivacyOptionsView.java]: Should probably be using `GeckoSession.LOAD_FLAGS_BYPASS_CACHE` as it has been fixed in GV.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/SitePermissionsOptionsView.java [hunk 2]
```diff
@@ -150,6 +160,19 @@ public void onChanged(List<SitePermission> sites) {
...
     };
+
+    private void reloadIfSameDomain(String aHost) {
+        if (mCategory != SitePermission.SITE_PERMISSION_WEBXR) {
+            return;
+        }
+        for (WindowWidget window: mWidgetManager.getWindows().getCurrentWindows()) {
+            Session session = window.getSession();
+            if (aHost.equalsIgnoreCase(UrlUtils.getHost(session.getCurrentUri()))) {
+                session.reload(GeckoSession.LOAD_FLAGS_NONE);
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/settings/SitePermissionsOptionsView.java]: And here too should use `GeckoSession.LOAD_FLAGS_BYPASS_CACHE`

**Comment by MortimerGoro** [comment]
@keianhzo Done

**Comment by keianhzo** [comment]
@MortimerGoro I don't see the items from my last comment fixed, maybe it's not pushed?

**Comment by keianhzo** [comment]
@MortimerGoro I see this fixed in the other PR so it's ok.

**Pull request merged**